src/common/config_validators.cc

```diff
@@ -9,6 +9,27 @@
 #include <vector>
 #include <boost/algorithm/string.hpp>
 #include <boost/lexical_cast.hpp>
+#include <boost/regex.hpp>
+
+int validate(md_config_t::option_rbd_default_pool_t *,
+             std::string *value, std::string *error_message) {
+  boost::regex pattern("^[^@/]+$");
+  if (!boost::regex_match (*value, pattern)) {
+    *error_message = "ignoring invalid RBD default pool";
+    *value = "rbd";
```

Comment:
May be we should somehow mention int the error message that 'rbd' is used instead? Because now when I have
```
% ./bin/rbd --rbd-default-pool '/sjsj' create test2 --size 128M 
parse error setting 'rbd_default_pool' to '/sjsj' (ignoring invalid RBD default pool)
```
It is not clear that the image is actually created in rbd pool.